source/comply/compliance-export.rst

```diff
@@ -80,13 +163,49 @@ Global Relay EML
    - For a **Custom** type, set the **Global Relay SMTP username**, **Global Relay SMTP password**, **Global Relay SMTP email address**, **SMTP Server Name**, and the **SMTP Server Port**, provided by Global Relay. **Custom** type can be used to integrate with Proofpoint.
 6. Select **Save**.
 
-Once you've selected Global Relay EML as your file format, you can set up an integration with Global Relay archive system. For more information, see `Global Relay Archive <https://www.globalrelay.com/products/archive-data-compliance/>`_.
-
 .. note::
   
    Messages larger than 250 MB will have their attachments removed because they are too large to send to Global Relay. An error is added to the server logs with id ``global_relay_attachments_removed``. It includes the post ID the attachments were removed from, as well as the attachment IDs. A `ticket is queued to better handle large messages <https://mattermost.atlassian.net/browse/MM-10038>`__.
 
-For more information on Global Relay archive system, see `their homepage <https://www.globalrelay.com/>`__.
+.. tab:: From Mattermost v10.5
+
+   You can review export job status in the System Console. Once you've selected Global Relay EML as your file format, you can set up an integration with Global Relay archive system.
+
+   When the daily compliance export job is finished, a parent directory is created named based on when the export was started and the ``startTimestamp`` and ``endTimestamp`` of the export, e.g, ``compliance-export-2024-08-13-05h08m-1723105062492-1723109100075``. That parent directory contains 1 zip file for each batch, named based on the batch number and the start and end timestamps of the messages in that batch, e.g, ``batch001-1723105062492-1723106622163.zip``. Each zip file contains the same information available in previous Mattermost server releases.
+
+   Working from the same example above, the directory would look like this:
+
+   .. code-block:: bash
+
+      compliance-export-2024-08-13-05h08m-1723105062492-1723109100075
+      ├── batch001-1723105062492-1723106622163.zip
+      ├── batch002-1723106622163-1723108196005.zip
+      └── batch003-1723108196005-1723109100075.zip
+
+   And each batch would look like this:
+
+   .. code-block:: bash
+
+      batch001-1723105062492-1723106622163.zip
+      ├── 20240808
+      └── actiance_export.xml
```

Comment:
This isn't relevant for Global Relay -- the results will be emailed to the recipient, at least one email per channel (or >1 email if the channel export is too large for one email).
We don't need to explain the format of the email, I think.